hy/core/bootstrap.hy

```diff
@@ -7,7 +7,7 @@
 ;;; They are automatically required everywhere, even inside hy.core modules.
 
 (defmacro if [&rest args]
-  "if with elif"
+  "Conditionally evaluate `alternating test and then expressions `args`."
```

Comment:
I think you've got a stray backtick and `args` here.

Reply:
Fixed it.